src/Command/Ide/IdeDeleteCommand.php

```diff
@@ -22,7 +23,7 @@ class IdeDeleteCommand extends IdeCommandBase {
    */
   protected function configure() {
     $this->setDescription('Delete a Cloud IDE')
-    ->addOption('cloud-app-uuid', 'uuid', InputOption::VALUE_REQUIRED, 'The UUID of the associated Cloud Platform Application');
+      ->addArgument('applicationUuid', InputArgument::OPTIONAL, 'The UUID or alias of the associated Cloud Platform Application');
```

Comment:
I wonder if we should instead have an optional argument for the Cloud IDE UUID here.